packages/fx-core/src/core/tools.ts

```diff
@@ -190,7 +190,11 @@ export function newEnvInfo(
     config: config ?? {
       azure: {},
       manifest: {
-        values: {},
+        values: {
+          appName: {
+            short: "myApp",
```

Comment:
Will the `myApp` be the default value for the manifest app name? or AppStudio plugin will update it to use the true application name (like the current toolkit's behavior)?

Reply:
This value will be updated in scaffolding and creating new environment.
![image](https://user-images.githubusercontent.com/71362691/132291605-3aa2cb7f-626c-4995-8cba-75c5a16be6c5.png)

Reply:
Thanks for explanation~